libUtils/win_firefox_webdriver.py

```diff
@@ -0,0 +1,111 @@
+from bs4 import BeautifulSoup
+import re
+import subprocess
+import platform
+import requests
+import zipfile
+import os
+
```

Comment:
cretae single file with name browse_manager, inside this create a base  class  BrowserManager  and your derived classes should be ChromeBrowserManager & FirefoxBrowserManager. 

BrowserManager   -->   Move common methods or attributes here 
ChromeBrowserManager --> specific to chrome
FirefoxBrowserManager  --> specific to firefox